.github/workflows/cml.yaml

```diff
@@ -0,0 +1,47 @@
+name: Generate CML Report
+ 
+on: [push]
+ 
+jobs:
+  generate_cml_report:
+    runs-on: ubuntu-latest
+    container: 
+      image: ghcr.io/moja-global/rest_api_gcbm:master
+      ports:
+        - "8080:8080"
+    steps:
+      - name: Remove GCBM output
+        id: remove_gcbm_output
+        run: |
+          rm -rf Standalone_GCBM/gcbm_project/output
+      - name: Install NPM # Need this for the CML setup
+        id: install_npm
```

Comment:
I think there's a standard action for that: 
```
actions/setup-node@v2
```

Reply:
Thanks a lot @aornugent! I wasn't aware 😅